### PR TITLE
fix: RUSTSEC-2023-0052

### PR DESCRIPTION
**Pull request by hanabi1224**
## Summary of changes

Changes introduced in this pull request:

Remove `webpki` crate from `Cargo.lock` by
- cargo update -p reqwest
- Remove `websocket` feature from `libp2p` as it does not seem to be useful at the moment

## Reference issue to close (if applicable)

Closes https://github.com/ChainSafe/forest/issues/3410

## Change checklist

- [x] I have performed a self-review of my own code,
- [x] I have made corresponding changes to the documentation,
- [x] I have added tests that prove my fix is effective or that my feature works (if possible),
- [x] I have made sure the [CHANGELOG][1] is up-to-date. All user-facing changes should be reflected in this document.

[1]: https://github.com/ChainSafe/forest/blob/main/CHANGELOG.md

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -8470,33 +8371,11 @@ dependencies = [
  "wasm-bindgen",
 ]
 
-[[package]]
-name = "webpki"
```
Comment (hanabi1224) [inline Cargo.lock:8474]: Problematic crate removed

File: src/libp2p/service.rs [hunk 2]
```diff
@@ -826,8 +826,7 @@ async fn emit_event(sender: &Sender<NetworkEvent>, event: NetworkEvent) {
 pub fn build_transport(local_key: Keypair) -> anyhow::Result<Boxed<(PeerId, StreamMuxerBox)>> {
     let build_tcp = || libp2p::tcp::tokio::Transport::new(libp2p::tcp::Config::new().nodelay(true));
     let build_dns_tcp = || libp2p::dns::TokioDnsConfig::system(build_tcp());
-    let transport =
-        libp2p::websocket::WsConfig::new(build_dns_tcp()?).or_transport(build_dns_tcp()?);
+    let transport = build_dns_tcp()?;
```
Comment (LesnyRumcajs) [inline src/libp2p/service.rs:829]: So, we are not using WS anymore? Do you think this is fine for communicating with other peers?

Comment (hanabi1224) [reply src/libp2p/service.rs:829]: WS is only useful when TCP is disabled or not available (e.g. WASM nodes that run in browsers), which is not the case (I don't think lotus offers an option to disable TCP) and none of the bootstrap nodes use WS address

Comment (LesnyRumcajs) [reply src/libp2p/service.rs:829]: Alright, in case it turns out we need it, we can re-enable it again.

**Review by LesnyRumcajs — APPROVED** [review]

**Review by lemmih — APPROVED** [review]

**Pull request merged**